src/cpp/middleware/fast/FastMiddleware.cpp

```diff
@@ -12,52 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <uxr/agent/middleware/FastMiddleware.hpp>
+#include <uxr/agent/middleware/fast/FastMiddleware.hpp>
 
 #include <fastrtps/xmlparser/XMLProfileManager.h>
-#include "../xmlobjects/xmlobjects.h"
+#include "../../xmlobjects/xmlobjects.h"
 
 namespace eprosima {
 namespace uxr {
 
 FastMiddleware::FastMiddleware()
 {
+    // TODO (Julian): implement DDS-XML parser.
     /* Load XML profile file. */
-    if (fastrtps::xmlparser::XMLP_ret::XML_OK != fastrtps::xmlparser::XMLProfileManager::loadDefaultXMLFile())
+    if (fastrtps::xmlparser::XMLP_ret::XML_OK != fastrtps::xmlparser::XMLProfileManager::loadXMLFile("./agent.refs"))
```

Comment:
Are you thinking to put this path as a cli option?

Reply:
Yeah, it was added to the CLI.